retropie_packages.sh

```diff
@@ -73,5 +73,5 @@ fi
 printMsgs "console" "${__INFMSGS[@]}"
 
 if [[ -n $__joy2key_pid ]]; then
-    kill -INT $__joy2key_pid 2>/dev/null
+    kill -INT $__joy2key_pid 2>/dev/null || echo "Failed to kill joy2key"
```

Comment:
> if this line causes it to stop due to being run with "exit on error" set, there are many other places which could fail - so I think this will need to be disabled via your automation software.

That sort of defeats the point of having an exit code then. In this particular case it's ok for joy2dev to not be running, because of the change I made above. Ignoring exit codes could mean that something breaks (which is actually broken), but everything will continue, and it will be almost impossible to debug.
I've tested this branch, and this is currently the only script that exists non-zero.

Would you be fine with this change instead:
```
if [[ -n $__joy2key_pid ]] && ps -p $__joy2key_pid; then
    kill -INT $__joy2key_pid 2>/dev/null
fi
View
```